src/components/ReplyComment/index.js

```diff
@@ -47,6 +47,7 @@ const ReplyCommentId = ({
     updateFeed,
     scrollViewRef,
     createComment,
+    onSaveHeight,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new function `onSaveHeight` is introduced but it's not clear from the provided context what this function does. Ensure that this function is properly defined and used in the right context.
```diff
-    onSaveHeight,
+    onSaveHeight, // Make sure this function is properly defined and used
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->